### ISSUE TITLE
You seem to be misunderstanding copyright and free software.

### ISSUE THREAD

**Issue by ian-kelling** [blaine/slocan-statement#1]
> it is available to be forked

Well, this document has no copyright license attached to it, so it
defaults to all rights reserved + the basic permissions that github
gives for people to view it on github. That means no one has the right
to modify your document under copyright. It seems like you might need to
learn some basics of how copyright works.

> We agree that software and standards written to support federated systems will be published with licenses and agreements that require adherence to this charter. 

That would imply creating a new copyright license which would be nonfree. See freedom 0, https://www.gnu.org/philosophy/free-sw.html.

**Comment by blaine** [comment]
@ian-kelling I appreciate your comment, but taking a snide tone is not helpful here. Sadly, I wish I could say it was unusual for someone from the FSF to engage in this way.

You're correct, I omitted to add a LICENSE. I will say that from the text, it is clear that it is permissively licensed. I appreciate that the FSF view is a legalistic one, rooted in Western discourse. No reasonable human, and no court would have ever allowed me to retroactively claim copyright over a text I distributed in public along with an ask to (paraphrasing) "please distribute and modify as you see fit" when I have said in public (again, paraphrasing) "take it, change it as much as you like, and don't worry about letting me know."

That said, for the avoidance of doubt, I have added an explicit cc0 license to the repository.

Commit: https://github.com/blaine/slocan-statement/commit/ff770fe5e15ce5f4602763e98bc4f23fc953cc22

For what it's worth, and I don't imagine we'll find much common ground here but I'll say it anyways, I am not ignorant of the FSF's definition of free software. I *disagree* with the FSF's view on "free software"; for example, even though the AGPL forces federated software to be open source, it does not guarantee the right to communicate. The FSF's perspective is one that centres software developers and excludes the many people who use and contribute to software from the questions of "freedom", especially in the context of networked systems.

Good day, sir.

**Comment by ian-kelling** [comment]
> taking a snide tone is not helpful here

I didn't intend to be snide in any way. What I tried and am trying to do is to carefully choose my words in a way that is honest and helpful and clear. I'm sorry I didn't come across that way to you in my first message.

> it is clear that it is permissively licensed

It wasn't clear to me. It is actually a very common confusion that people write software without adding a free license and assume other people have permissions that they don't have. Most people aren't taught how copyright works. For a work to be "clearly permissively licensed", the I would expect it be under a license listed here https://www.gnu.org/licenses/license-list.html as permissive.

> No reasonable human, and no court would have ever allowed me to retroactively claim copyright over a text I ...

Saying "allow you to retroactively claim copyright" again seems like a misunderstanding. Every work has a copyright as soon as it is created and it is automatically held by the person who created the work unless they have some other agreement or it is a derivative work. No one else has rights to do things with the work that copyright prohibits unless they have a license to do so. The previous statement you had, "I encourage the use of the #SlocanStatement hashtag on the Fediverse if you post a derivative work elsewhere." was not a license I would want to depend on.

I don't agree with your characterization of the FSF's views or how the FSF engages with others. My comment was to help you do what you seemed to intend, giving people freedom over your work, which I found interesting enough to help you with.
